proj/bq/test/proj_integration.js

```diff
@@ -0,0 +1,59 @@
+const assert = require('assert').strict;
+const chaiAssert = require('chai').assert;
+const {BigQuery} = require('@google-cloud/bigquery');
+
+const BQ_PROJECTID = process.env.BQ_PROJECTID;
+const BQ_DATASET_PROJ = process.env.BQ_DATASET_PROJ;
+
+describe('PROJ integration tests', () => {
+
+    let client;
+    before(async () => {
+        if (!BQ_PROJECTID) {
+            throw "Missing BQ_PROJECTID env variable";
+        }
+        if (!BQ_DATASET_PROJ) {
+            throw "Missing BQ_DATASET_PROJ env variable";
+        }
+        client = new BigQuery({projectId: `${BQ_PROJECTID}`});
+    });
+
+    it ('Returns the proper version', async () => {
+        const query = `SELECT \`${BQ_PROJECTID}\`.\`${BQ_DATASET_PROJ}\`.VERSION() as versioncol;`;
+        let rows;
+        await assert.doesNotReject( async () => {
+            const [job] = await client.createQueryJob({ query: query });
+            [rows] = await job.getQueryResults();
+        });
+        assert.equal(rows.length, 1);
+        assert.equal(rows[0].versioncol, 1);
+    });
+
+    it ('Proj method should work', async () => {
```

Comment:
I'm missing some tests with invalid data (invalid projections, invalid/incomplete coordinates) and checks to confirm this works with arrays > 1.